buildSrc/src/main/groovy/bio.terra.catalog.java-common-conventions.gradle

```diff
@@ -41,15 +41,15 @@ repositories {
 
 dependencies {
     compileOnly 'com.github.spotbugs:spotbugs-annotations:4.7.2'
-    implementation 'io.swagger.core.v3:swagger-annotations:2.2.0'
-    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.31'
-
+    implementation 'io.swagger.core.v3:swagger-annotations:2.2.12'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
     implementation 'org.slf4j:slf4j-api'
 
     testImplementation 'ch.qos.logback:logback-classic'
 
-    implementation 'bio.terra:terra-common-lib:0.0.93-SNAPSHOT'
-    implementation 'bio.terra:datarepo-client:1.349.0-SNAPSHOT'
+    implementation 'bio.terra:terra-common-lib:0.1.9-SNAPSHOT'
+    implementation 'com.google.guava:guava:31.1-jre'
```

Comment:
Without it I get this error: 
```
Could not find com.google.guava:guava:.
Required by:
    project :client > bio.terra:terra-common-lib:0.1.9-SNAPSHOT:20231212.164923-1
```

Reply:
This one puzzles me too. It's similar to the issue we were seeing with spring-web-aop. We're depending on terra-common-lib, which depends on guava, but we shouldn't need to depend on guava directly. We should be able to get it from a transitive dependency. It would be nice to better understand what's going on here.